packages/jumpstarter-driver-can/README.md

```
@@ -18,17 +25,120 @@
+        txid: 2
+      params:
+        max_frame_size: 2048
+        blocking_send: false
+        can_fd: true
+
+```
+
+| Parameter     | Description | Type | Required | Default |
+| --------------| ----------- | ---- | -------- | ------- |
+| channel       | can bus to be used i.e. `vcan0`, `vcan1`, etc.. |  `str` | yes |  |
+| address       | Refer to the [isotp.Address](https://can-isotp.readthedocs.io/en/latest/isotp/addressing.html#isotp.Address) documentation | isotp.Address | yes |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)